packages/framework/src/Resources/translations/messages.cs.po

```diff
@@ -1297,6 +1321,12 @@ msgstr "Editace zákazníka"
 msgid "Editing customer - %name%"
 msgstr "Editace zákazníka - %name%"
 
+msgid "Editing customer user role group"
+msgstr "Editace skupiny práv zákazníka"
```

Comment:
we should unify the translation of customer user role group. I see two different translations: `skupina práv zákazníka`, `skupina uživatelských práv`, and `skupina práv`

Reply:
all translations fixed here: https://github.com/shopsys/shopsys/pull/3323/commits/0b690ff3e88fe5ddc262451e058eaba25effc2c2